### DISCUSSION TITLE
When is the next release?

### DISCUSSION THREAD

**Discussion by VictorioBerra** [ASP-NET-MVC-Boilerplate/Framework#26] [Q&A]

**Comment by VictorioBerra** [comment]
@RehanSaeed can you push this out to NuGet?

**Comment by RehanSaeed** [comment]
Is there something in particular you want from a release?

**Comment by VictorioBerra** [comment]
The bug fix to one of the authorize operation filters.

**Comment by VictorioBerra** [comment]
https://github.com/Dotnet-Boxed/Framework/pull/25

**Comment by RehanSaeed** [comment]
Releasing now.
